frontend/src/app/components/dashboard/MyPoliciesRow.tsx

```diff
@@ -1,7 +1,5 @@
-import React from 'react'
+import React from "react";
```

Comment:
Rip, im a single quotes kinda guy but whatever this is cool by me as long as we're consistent (which the new settings file should enforce)